android/app/src/main/res/xml/prefs_main.xml

```diff
@@ -97,6 +97,12 @@
       android:summary="@string/enable_show_on_lock_screen_description"
       android:defaultValue="true"
       android:order="17"/>
+    <Preference
+        android:key="@string/pref_map_locale"
+        android:title="@string/pref_map_locale_title"
+        app:singleLineTitle="false"
+        android:persistent="false"
+        android:order="18"/>
```

Comment:
IS this option less important than other options? Should it be in a different position?